### DISCUSSION TITLE
Install Git LFS on OpenSuse

### DISCUSSION THREAD

**Discussion by polgarriga** [git-lfs/git-lfs#5022] [Q&A]
I need to install Git LFS on OpenSUSE.

I have downloaded the `script.rpm.sh` file by running `curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | sudo bash`, as shown in [here](https://github.com/git-lfs/git-lfs/blob/main/INSTALLING.md).

Next, they tell me I run `sudo yum install git-lfs` on my Terminal, but I get: `sudo: yum: command not found`.

**Comment by bk2204** [comment]
Hey,

It may be that you're using `dnf` instead and should specify `sudo dnf install git-lfs`.  Unfortunately, I fear that I'm not very familiar with RPM-based distributions, so if that doesn't work, I'm not sure how to get it to work.

**Reply by polgarriga** [reply]
No, it says the same thing: sudo: dnf: command not found. Let's see if someone else can give me a hand.

**Comment by hudson-newey** [comment]
I know this is an old thread, but for anyone who comes across this in the future, do not use the PackageCloud script, run the following command from your terminal:

```sh
$ sudo zypper install -y git-lfs && git lfs install
```

OR run the following commands line by line

```sh
$ sudo zypper install git-lfs
```
```sh
$ git lfs install
```
